### PR TITLE
Fix cuda version

### PR DESCRIPTION
**Pull request by unnonouno**
I found some combinations of cuda and cuda related libraries do not work. I replaced them.

### CODE REVIEW

**Comment by unnonouno** [comment]
jenkins test this please

File: run_test.py [hunk 1]
```diff
@@ -90,7 +90,7 @@
         conf = {
             'base': 'ubuntu14_py2',
             'cuda': 'cuda90',
-            'cudnn': 'cudnn5',
+            'cudnn': 'cudnn-cuda9',
```
Comment (okuta) [inline run_test.py]: Is this correct?

**Comment by unnonouno** [comment]
Sorry, i fixed it.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**